packages/block-library/src/query-total/index.php

```diff
@@ -23,15 +23,15 @@ function render_block_core_query_total( $attributes, $content, $block ) {
 	$wrapper_attributes = get_block_wrapper_attributes();
 	if ( isset( $block->context['query']['inherit'] ) && $block->context['query']['inherit'] ) {
 		$query_to_use = $wp_query;
-		$current_page = max( 1, get_query_var( 'paged', 1 ) );
+		$current_page = max( 1, (int) get_query_var( 'paged', 1 ) );
```

Comment:
The `get_query_var` also uses `WP_Query::get` under the hood and ``min( 1, '0' )` will return `"0"`.